Runtime error

Hi folks:
We rebooted our cluster and have started seeing this error with singularity run  command with version 3.5 and 3.6:
```
panic: runtime error: index out of range [9] with length 9
goroutine 7 [running]:
github.com/sylabs/singularity/pkg/util/fs/proc.parseMountInfoLine(0xc0000d86e0, 0x43, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        github.com/sylabs/singularity/pkg/util/fs/proc/proc.go:113 +0x581
github.com/sylabs/singularity/pkg/util/fs/proc.GetMountInfoEntry(0xab9890, 0x14, 0x0, 0x0, 0x0, 0x0, 0x0)
        github.com/sylabs/singularity/pkg/util/fs/proc/proc.go:145 +0x1f6
github.com/sylabs/singularity/internal/pkg/runtime/engine/singularity.(*EngineOperations).prepareAutofs(0xc0000fefd0, 0xc000010678, 0x0, 0xab019c)
        github.com/sylabs/singularity/internal/pkg/runtime/engine/singularity/prepare_linux.go:399 +0x4b
github.com/sylabs/singularity/internal/pkg/runtime/engine/singularity.(*EngineOperations).prepareContainerConfig(0xc0000fefd0, 0xc000010678, 0x0, 0x0)
        github.com/sylabs/singularity/internal/pkg/runtime/engine/singularity/prepare_linux.go:596 +0x8a7
github.com/sylabs/singularity/internal/pkg/runtime/engine/singularity.(*EngineOperations).PrepareConfig(0xc0000fefd0, 0xc000010678, 0x5, 0xab7415)
        github.com/sylabs/singularity/internal/pkg/runtime/engine/singularity/prepare_linux.go:128 +0x5d8
github.com/sylabs/singularity/internal/app/starter.StageOne(0xc000010678, 0xc00000f340)
        github.com/sylabs/singularity/internal/app/starter/stage_linux.go:27 +0x8c
main.startup()
        command-line-arguments/main_linux.go:56 +0x2fb
created by main.main
        command-line-arguments/main_linux.go:102 +0x35
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
-------------------------------------------------------------------------
```

--- a/pkg/util/fs/proc/proc.go
+++ b/pkg/util/fs/proc/proc.go
@@ -83,7 +83,7 @@ type MountInfoEntry struct {
 // a MountInfoEntry containing parsed fields associated
 // to the line.
 func parseMountInfoLine(line string) MountInfoEntry {
-	fields := strings.Fields(line)
+	fields := strings.Split(line, " ")
 	entry := MountInfoEntry{}
 
 	// ID field

